Makefile

```diff
@@ -23,21 +23,46 @@ build-release-arm64:
 	CGO_ENABLED=0 GOOS=linux   GOARCH=arm64 go build ${LDFLAGS} -o=fortigate-exporter.linux.arm64 .        && \
  	CGO_ENABLED=0 GOOS=darwin  GOARCH=arm64 go build ${LDFLAGS} -o=fortigate-exporter.darwin.arm64 .
 
+.PHONY: generate-html-coverage
+generate-html-coverage:
+	go tool cover -html=cover.out -o coverage.html
+	@printf "Generated coverage html \n"
+
+.PHONY: print-coverage
+print-coverage:
+	@go tool cover -func cover.out
+
+.PHONY: test-unittests
+test-unittests:
+	go test -v -race -coverprofile cover.out ./...
+
 .PHONY: test
-test:
-	go test -v -race ./...
+test: fmt-check vet test-unittests generate-html-coverage print-coverage
+	@printf "Sucessfully run tests \n"
+
+.PHONY: test-update-snapshots
```

Comment:
This is for a later PR no?

Reply:
Yeah, that's correct. This is for the PR which reworks the config handling